apps/search/js/search.js

```diff
@@ -246,6 +261,16 @@
     },
 
     /**
+     * Called to notify the AppUsageMetrics of a search
+     */
+    metricsNotify: function(provider) {
+      this._portMetrics.postMessage({
```

Comment:
is it possible for `this._portMetrics` to be uninitialized when `metricsNotify` is called?